src/Microsoft.AspNet.Authentication/AuthenticationHandler.cs

```diff
@@ -104,6 +104,7 @@ protected string CurrentUri
                 if (ticket?.Principal != null)
                 {
                     Context.User = SecurityHelper.MergeUserPrincipal(Context.User, ticket.Principal);
+                    Logger.LogInformation(0, $"HttContext.User merged via AutomaticAuthentication from authenticationScheme: {Options.AuthenticationScheme}.");
```

Comment:
You know, come to think of it, I think these aren't supposed to use string interpolation or format strings.

Logging has its own format stuff for structured data, something like:

```c#
Logger.LogInformation(0, "HttContext.User merged via AutomaticAuthentication from authenticationScheme: {AuthenticationScheme}.", Options.AuthenticationScheme);
```

Note that there's no string formatting going on here, but Logging will see the `{foo}` in the string and create an associated structured data element with `AuthenticationScheme = Options.AuthenticationScheme`.

I missed this in my earlier logging change.